rfb/rfb.h

```diff
@@ -763,9 +763,8 @@ extern rfbBool rfbSetNonBlocking(int sock);
 /* websockets.c */
 
 extern rfbBool webSocketsCheck(rfbClientPtr cl);
-extern rfbBool webSocketCheckDisconnect(rfbClientPtr cl);
-extern int webSocketsEncode(rfbClientPtr cl, const char *src, int len, char **dst);
-extern int webSocketsDecode(rfbClientPtr cl, char *dst, int len);
+extern int webSocketsWrite(rfbClientPtr cl, const char *src, int len);
```

Comment:
Pls don't change public API here, i.e the functions should retain their names and webSocketCheckDisconnect be made a NOP (but the symbol should still be there to not break backwards compatibility)